io-engine/src/bin/io-engine-client/v1/snapshot_cli.rs

```diff
@@ -11,6 +11,7 @@ use colored_json::ToColoredJson;
 use mayastor_api::v1 as v1_rpc;
 use snafu::ResultExt;
 use tonic::Status;
+use v1_rpc::snapshot::*;
```

Comment:
Try to avoid these imports and use v1_rpc prefix.

Reply:
fixed the code.